cmake/trailbook/add-trailbook.cmake

```diff
@@ -0,0 +1,705 @@
+
+# This macro is for internal use only
+#
+# It is used in the function add_trailbook.
+# It checks the requirements defined by the requirements.txt file
+# and installs any missing packages into the current Python virtual environment.
+# It checks during the configuration phase.
+macro(_add_trailbook_check_requirements_txt)
+    if(EXISTS ${args_REQUIREMENTS_TXT})
+        execute_process(
+            COMMAND
+                ${Python3_EXECUTABLE}
+                ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/check_requirements_txt.py
+                ${args_REQUIREMENTS_TXT}
+                --fix-in-venv
+            RESULT_VARIABLE _CHECK_REQUIREMENTS_TXT_RESULT
+            COMMAND_ERROR_IS_FATAL NONE
```

Comment:
This fails for me on cmake `3.28.3`: https://cmake.org/cmake/help/v3.28/command/execute_process.html
COMMAND_ERROR_IS_FATAL only allows `ANY` or `LAST`. I think you simply drop `COMMAND_ERROR_IS_FATAL NONE` since you are checking the result afterwards anyways